src/status_im/ui2/screens/chat/group_details/view.cljs

```diff
@@ -0,0 +1,96 @@
+(ns status-im.ui2.screens.chat.group-details.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [status-im.ui2.screens.chat.group-details.style :as style]
+            [quo2.core :as quo2]
+            [utils.re-frame :as rf]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.ui.components.list.views :as list]
+            [status-im.ui2.screens.chat.components.contact-item.view :as contact-item]))
+
+(defn back-button []
+  [quo2/button {:type                :grey
+                :size                32
+                :width               32
+                :style               {:margin-left 20}
+                :accessibility-label :back-button
+                :on-press            #(rf/dispatch [:navigate-back])}
+   [quo2/icon :i/arrow-left {:color (colors/theme-colors colors/neutral-100 colors/white)}]])
+
+(defn options-button []
+  [quo2/button {:type                :grey
+                :size                32
+                :width               32
+                :style               {:margin-right 20}
+                :accessibility-label :options-button}
+   [quo2/icon :i/options {:color (colors/theme-colors colors/neutral-100 colors/white)}]])
+
+(defn count-container [count]
+  [rn/view {:style (style/count-container)}
+   [quo2/text {:size   :label
+               :weight :medium
+               :style  {:text-align :center}} count]])
+
+(defn prepare-members
+  [contact-addresses admins]
+  (let [contacts (map #(assoc (rf/sub [:contacts/contact-by-address %])
+                              :admin? (get admins %))
+                      contact-addresses)
+        admins   (filter :admin? contacts)
+        online   (filter #(and (not (:admin? %)) (:online? %)) contacts)
+        offline  (filter #(and (not (:admin? %)) (not (:online? %))) contacts)]
+    (vals (cond-> {}
+            (seq admins) (assoc :owner {:title :Owner :data admins})
```

Comment:
just wondering why Owner and Online big O

Reply:
That's the design. Capital letters.

Reply:
i mean its a keyword, how does it work then ?

Reply:
We cannot have capital letters in keywords? IDK it works!

Reply:
This data goes into a SectionList which is a RN component. So it gets serialized to JS I think, so I guess that's why it works

Reply:
I found this SO quite informative https://stackoverflow.com/questions/3951761/what-are-the-allowed-characters-in-a-clojure-keyword

Reply:
So you want me to change it to string?

Reply:
yeah not sure why the keyword was used